### ISSUE TITLE
<Table> height increase

### ISSUE THREAD

**Issue by nicolas2bert** [scality/core-ui#170]
**Component**:

Table

**What happened**:

Using Table component  height increases

**What was expected**:

Stable height

**Steps to reproduce**
 
Import and use Table component. 
It seems to also happen here: https://scality.github.io/core-ui/?path=/story/table--default

**Resolution proposal** (optional):

**Comment by ChengYanJin** [comment]
We met and fixed this bug in the metalk8s side, you can find how to solve this issue from react-virtualized table official documentation.

**Comment by thomasdanan** [comment]
We should close this one as soon as the new table component is available in core-ui

**Comment by nicolas2bert** [comment]
Yes, we can close this one as soon as the Table component is removed from core-ui.
It could be replaced later.
